src/components/menubar/MenuShell.tsx

```diff
@@ -5,7 +5,9 @@ import { useTheme } from '__/hooks/use-theme';
 import type { TTheme } from '__/stores/theme.store';
 import { theme } from '__/theme';
 
-interface IMenuShell {}
+interface IMenuShell {
+  children: React.ReactNode;
```

Comment:
This isn't needed really. React's `FC` automatically injects the `children` field.